src/result_gen.py

```diff
@@ -0,0 +1,563 @@
+        artifact_result = Bunch()
+        artifact_result.artifact_id = artifact[0]
+        artifact_result.path = '/'.join(artifact[3].split('/')[4:])
+        artifact_result.prediction = str(prediction[0])
+        results.append(artifact_result)
+    return results
+
+
+def move_processed_data(json_path, table, state):
+    # destination_folder1 = destination_folder + "{0}_{1}".format(state, table)
+    destination_folder = '/'.join(json_path.split('/')[:-3])
+    destination_folder = destination_folder + \
+        "/" + "{0}_{1}".format(state, table)
+    if not os.path.exists(destination_folder):
+        os.makedirs(destination_folder)
+    try:
+        filename = json_path.split('/')[-1]
+        shutil.move(json_path, os.path.join(destination_folder, filename))
+    except Exception as error:
+        print(error)
+
+
+def get_file_name(destination_folder, model_id, qr_code, environment):
+    filename = '{0}-{1}-{2}-{3}.json'.format(
+        environment, model_id, qr_code, random.randint(10000, 99999))
+    folder = '{0}/{1}/'.format(destination_folder, model_id)
+    return filename, folder
+
+
+def extract_status_code_one(path):
+    '''
+    Extract the status code from the artifact path
+    '''
+    status_code = path.split('/')[-1].split('_')[-2]
+    return status_code
+
+
+def extract_status_code_two(path):
+    '''
+    Extract the status code from the artifact path
+    '''
+    status_code = path.split('/')[-1].split('_')[-3]
+    return status_code
+
+
+def check_status_code(path, extract_status_code, status_list):
+    '''
+    Check the artifact belong to one of the status code in status list
+    '''
+
+    if extract_status_code(path) in status_list:
+        return True
+    else:
+        return False
+
+
+class Measure_Result_Generation:
+
+    def __init__(
+            self,
+            measure_id,
+            main_connector,
+            replace_path,
+            container_name):
+        self.measure_id = measure_id
+        self.main_connector = main_connector
+        self.container_name = container_name
+
+        if rgutils.depth_exists(measure_id, self.main_connector):
+            self.dataformat = 'depth'
+        else:
+            self.dataformat = 'pcd'
+
+        self.replace_path = replace_path
+        self.container_name = container_name
+        self.front_status_list = ['100', '104', '200']
+        self.back_status_list = ['100', '104', '200']
+        self.threesixty_status_list = ['100', '104', '200']
+        self.ACC_NAME = config.ACC_NAME
+        self.ACC_KEY = config.ACC_KEY
+
+    def get_artifact_list_per_measure(self):
+        '''
+        Get the list of artifact per measure
+        '''
+        get_artifacts = "SELECT id, qr_code, create_timestamp, replace('{}'".format(
+            self.replace_path)
+        get_artifacts += " || split_part(storage_path,'/storage/emulated/0/Child Growth Monitor Scanner App/', 2), 'measurements', 'measure')"
```

Comment:
What is `/storage/emulated/0/Child Growth Monitor Scanner App/`. 
Consider putting it into a global variable